lib/modules/tooltip.js

```diff
@@ -278,8 +278,18 @@ export class _TooltipTarget extends React.Component {
         onMouseLeave: this.handleCloseTooltip,
       };
     } else if (type === TARGET_TYPES.CLICK) {
+      var open = false;
+
       return {
-        onClick: this.handleOpenTooltip,
+        onClick: function onClick(e) {
+          if (!open) {
+            _this3.handleOpenTooltip(e);
+            open = true;
+          } else {
+            _this3.handleCloseTooltip(e);
+            open = false;
+          }
+        }
```

Comment:
Might be cleaner to have another method called something like `handleTooltip` that encapsulates this logic. Then we can just do something like:
```
return {
    onClick: this.handleTooltip(e, open);
}
```